src/libs/actions/IOU.js

```diff
@@ -215,6 +215,14 @@ function setMoneyRequestMerchant_temporaryForRefactor(transactionID, merchant) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {merchant: merchant.trim()});
 }
 
+/**
+ * @param {String} transactionID
+ * @param {Object} pendingFields
+ */
+function setMoneyRequestPendingFields(transactionID, pendingFields) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {pendingFields});
```

Comment:
Shouldn't the pending fields get set under `transactionDraft.transactionID.pendingFields` vs just `transactionDraft.transactionID`?

Reply:
It is setting the `transactionDraft.transactionID.pendingFields`, I'm just using the shorthand property here. This can be read as:

```js
Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {pendingFields: pendingFields});
```

Reply:
Oh yeah right haha. Wow I guess I haven't read enough javascript lately 😅